src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -188,15 +188,18 @@ static pwmOutputPort_t pwmGyroClk = {0};
 
 static bool initExternalClock(const extDevice_t *dev)
 {
-    int cfg;
-    if (&gyro.gyroSensor1.gyroDev.dev == dev) {
-        cfg = 0;
-    } else if (&gyro.gyroSensor2.gyroDev.dev == dev) {
-        cfg = 1;
-    } else {
-        // only gyroSensor<n> device supported
+    int cfg = -1;
```

Comment:
Maybe `static int findByExtDevice(const extDevice_t *dev) { `

Reply:
I'm not sure what you mean here.